pkg/kubelet/cadvisor/cadvisor_windows.go

```diff
@@ -58,6 +58,10 @@ func (cu *cadvisorClient) ContainerInfoV2(name string, options cadvisorapiv2.Req
 	return cu.winStatsClient.WinContainerInfos()
 }
 
+func (cu *cadvisorClient) GetRequestedContainersInfo(containerName string, options cadvisorapiv2.RequestOptions) (map[string]*cadvisorapi.ContainerInfo, error) {
+	return nil, nil
```

Comment:
Why is this also not errUnsupported?

Reply:
This PR switches from using SubcontainerInfo to GetRequestedContainersInfo.  The only difference between the two is that SubcontainerInfo takes a request, and GetRequestedContainersInfo takes options.  This is maintaining the same behavior as SubcontainerInfo, which currently returns `nil, nil` as well.

Reply:
Thanks for explaining :+1: